doc/whatsnew/2.9.rst

```diff
@@ -39,6 +39,11 @@ New checkers
 
 * New checker ``await-outside-async``: Emitted when await is used outside an async function.
 
+* Add new extension ``CodeStyleChecker``. It includes checkers that can improve code
+  consistency. As such they don't necessarily provide a performance benefit
```

Comment:
But this one provide a small performance benefit right ?

Reply:
Yeah it does, although it's probably a negligible aside from some edge cases.

I don't think it would make sense to create yet another extension just for such cases. We should think about this more in  these categories:
1. Does a checker provide a clear benefit, i.e. detect a common issue or improve performance => it should probably be part of the core checker classes
2. Is it something that would improve code consistency, maybe because it's slightly better with regards to performance and therefore preferred => the new extension is probably the right place
3. Everything else should go into another extension

The `CodeStyleChecker` extension shouldn't be the catch all one for everything that isn't good enough for the core. Especially if we want devs to actually use it.

Reply:
> The CodeStyleChecker extension shouldn't be the catch all one for everything that isn't good enough for the core. Especially if we want devs to actually use it.

Makes perfect sense !